### DISCUSSION TITLE
Make nuxt-auth work with i18n

### DISCUSSION THREAD

**Discussion by lowbits** [Teranode/nuxt-module-alternatives#100] [Ideas]
Hey, 
maybe an old discussion, but it was also a problem in the old nuxt-auth module.
The redirects don't respect localePath from i18n.

So if you have

```
redirects: {
  home: "/home",
  login: 'login'
}

```
and your current URL looks like this:

`http://localhost:3000/en/home`

And you get redirected through a middleware, the URL will look like:

`http://localhost:3000/home`

Maybe you have an idea on how to fix it easy.

[See old discussion](https://github.com/nuxt-community/auth-module/discussions/1538)

Edit:
Currently i have this workaround

```ts
// ./plugins/auth-redirects/index.ts
import {defineNuxtPlugin} from "#app";

export default defineNuxtPlugin(({$auth, $localePath}) => {
    $auth.onRedirect((to:string) => {
        return $localePath(to)
    })
})
```

**Comment by Denoder** [comment]
I'll try to see if I can integrate it later today.
